CONTRIBUTING.md

```diff
@@ -99,27 +104,42 @@ Example of a module directory:
   
 *  GitHub doesn't support embedded videos, so if you need that please just insert it as a regular hyperlink (or hyperlinked thumbnail image of the video). 
 
-### General Advice
+### Github Pages
 
-* As the content generation process evolves, some contributions will undoubtedly stand out as exemplary. Don't be shy about copying those in style, syntax, etc.
+Here is a brief overview of how files and directories should be organized. Explicitly named files or directories should remain fixed in location and name, and assumed to exist as such by other files:
+
+```bash 
+HPC  # i.e. the root of this repo
+├── docs
+    ...
+    └── Documentation      # This directory is where documentation contributions should go. 
+        ...
+        └── <...>      # Modules that exist or will exist
+            ...
+            ├─ index.md  # Optional overview page for the content section.
+            ...
+            └ ... ─ <...>  # Sub-modules that exist or will exist.
+├── overrides  # This directory should contain images.
+├── mkdocs.yml # This file contains configuration settings for the site.
+├── README.md  # The homepage of the repository.
+```
+* If your documentation fits into one of the categories currently in the pages, you may place it in the coresponding directory. Otherwise, make a new directory in Documentation. 
+* **Files must be listed in the mkdocs.yml nav section in order to be rendered.** The nav section dictates the navigation bar structure on the site. Section headers must be included followed by an indented list of the paths to the files the section should contain. Please see example below. 
+```bash 
+ - Documentation:
+      - Data and File Systems:
+        - File Systems:
+          - Documentation/Data-and-File-Systems/File-Systems/index.md
+          - Lustre:
+            - Documentation/Data-and-File-Systems/File-Systems/Lustre/lustrebestpractices.md
+```          
+* If you would like the section header in the navigation bar to be directly linked to documentation (e.g. an overview page), place this content in an index.md file in the respective folder, and add it to the beginning of its nav section in the mkdocs.yml file. 
+* Images should be placed in the overrides directory and linked to with the relative path without the overrides directory included like ```../../../assets/images/conda_logo.png``` **not** ```../../../overrides/assets/images/conda_logo.png```.
+
+*  Any links to internal files must be the relative path to it from the referencing file, not the absolute path. 
+*  Code blocks will be automatically highlighted when rendered on the website, so include them or `inline code` wherever appropriate instead of terminal screenshots.
```

Comment:
* Code blocks will be automatically highlighted when rendered on the website, so please either use a code block or `inline code` whenever practical instead of a screenshot of a terminal.